extensions/hibernate-reactive/deployment/pom.xml

```diff
@@ -72,6 +103,117 @@
                     </annotationProcessorPaths>
                 </configuration>
             </plugin>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
         </plugins>
     </build>
+
+    <profiles>
```

Comment:
Just nit-picking, but if it's an end-to-end integration test does it not belong among the other integration tests?

It's not a problem but I wonder how far it helps to reduce start/stopping of databases; also it seems unusual to need to start containers when one just wants to run the tests of the deployment module.

Reply:
I'm doing the same thing we're currently doing with Hibernate ORM. It so happens that ORM can use H2 and thus doesn't need a container, but Reactive doesn't support any in-memory database.

FWIW, this is not a first. We're doing this for Hibernate Search too, where we use a container for Elasticsearch.

The main problem with tests in the `integration-tests` directory is that, as far as I can tell, those only allow a single configuration common to all tests. I'm anticipating more configurations for other tests here (such as multi-PU tests, like we do for ORM).

There's also the fact that while we're using PostgreSQL here, it actually doesn't matter which one we use. So it would feel wrong to put this test in `hibernate-reactive-postgresql` and not in `hibernate-reactive-mysql`. And also wrong to copy/paste it in every project.